packages/core/src/QueryBuilder/RuleGroup/RuleGroup.tsx

```diff
@@ -198,7 +197,10 @@ export const RuleGroup = ({
                 onClick={() => {
                   dispatchAction({ type: "add-rule", id });
                 }}
-                className={classes.createConditionButton}
+                className={cx(classes.createConditionButton, {
+                  [classes.createButtonDisabled]: readOnly,
+                })}
```

Comment:
Can we use the `:disabled` selector instead of adding new classes?

Reply:
I did that but the tests were breaking. For some reason, the `pointerEvents: "none"` style was applied even if the button was not disabled. I'll try again though.

Reply:
Updated ✅